third_party/move/move-compiler-v2/tests/checking/variants/variants_ok.move

```diff
@@ -0,0 +1,116 @@
+module 0x815::m {
+    use 0x815::m;
+
+    enum Color {
+        RGB{red: u64, green: u64, blue: u64},
+        Red,
+        Blue,
+    }
+
+    enum ColorUsesBlockNoComma {
+        RGB{red: u64, green: u64, blue: u64}
```

Comment:
If you don't have a language spec for your feature you are implementing, it would be very useful to provide references to related language specs you are emulating.  I was unfamiliar with this Rust feature.

Reply:
There is a design doc and there was a design review. The doc states, and it was discussed in the review, that we are orienting towards Rust.

Having said so, I can imagine we make the comma mandatory. FWIW, the Rust formatter always adds a comma (you need to enable this via an option, and in aptos-core, we have enabled this option). There is a merit to make the syntax canonical. Its really just not purpose of this PR to finalize such nitty-gritty details.